easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -270,8 +326,51 @@ def configure_step(self):
             raise EasyBuildError("Failed to move non-espresso directories: %s", err)
 
     def install_step(self):
-        """Skip install step, since we're building in the install directory."""
-        pass
+        """Custom install step for Quantum ESPRESSO."""
+
+        # Copy all binaries
+        try:
+            # binary
+            bindir = os.path.join(self.installdir, 'bin')
+            shutil.copytree(os.path.join(self.cfg['start_dir'], 'bin'), bindir)
```

Comment:
@akesandgren Please use `copy_dir` and `cope_file` from `filetools` rather than using `shutil.*` directly.

if you do, there's no need anymore to wrap this in a `try`/`except`...